Assets/Scenes/SubBoom.cs

```diff
@@ -285,26 +370,33 @@ void Update()
             foreach (var sub in submarines)
             {
                 //this isn't working for some reason? not even the debug message is showing up.
-                if (ec.explodeCharge.GetComponent<BoxCollider2D>().IsTouching(sub.submarine.GetComponent<BoxCollider2D>()) == true)
+                if (ec.explodeCharge.GetComponent<BoxCollider2D>().IsTouching
+                   (sub.submarine.GetComponent<BoxCollider2D>()) == true)
                 {
                     Debug.Log("This works!");
+
                     Destroy(sub.submarine);
                     submarines.Remove(sub);
+
+                    score += 1;
```

Comment:
I kind of liked giving the player a point for ever Update frame they're moving the ship around. It doesn't amount to much but it's kind of entertaining to see the score ticking upward. I was thinking that killing a submarine would be worth like 1000 or 10,000 points. Scores might be depressingly low if we only give out a point per sub :) What do you think? It seems like something we can make up however it seems most "fun" :)

Reply:
i feel if the score increases rapidly often it won't feel as satisfying to see your score increase in general or something that's important. maybe we could add a point for every second, or 100 points every 10 seconds? then whenever we hit a submarine, the score could increase 100 points or 1000 points?

Reply:
Sitting with it some more, maybe we should just try the one point per submarine scoring and see how we like it.